pkg/api/fuzzer/fuzzer.go

```diff
@@ -439,6 +439,14 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 					ss.Ports[i].TargetPort.StrVal = "x" + ss.Ports[i].TargetPort.StrVal // non-empty
 				}
 			}
+			if ss.SessionAffinity == api.ServiceAffinityClientIP && ss.SessionAffinityConfig == nil {
```

Comment:
we probably should fuzz affinity explicitly.  There are many examples to follow, but basically declare a slice of valid values, pick a random number mod the length of that slice, set the value and THEN do this fixup.  Basically every enumerated field should go through this.

Reply:
The chance of the fuzzer choosing a valid string is ~0

Reply:
ACK. Please see if the changes are fine.

PS. Since there is a restriction on affinity type and affinity config - config is required when type is ClientIP. So, I  employ a temporary struct,

```
struct {
	affinityType   api.ServiceAffinity
	affinityConfig *api.SessionAffinityConfig
}
```

to describe the relationship between them.